util/module_logger.go

```diff
@@ -0,0 +1,260 @@
+package util
+
+import (
+	"fmt"
+	log "github.com/uber-common/bark"
+)
+
+// The level definition is stolen from logrus.
```

Comment:
The logrus levels are in reverse order and don't implement OffLevel. The problem with the reverse order is that an uninitialized level defaults to PanicLevel.